daily_word_bot/app.py

```diff
@@ -148,12 +214,17 @@ def send_word(self, context: CallbackContext):  # pragma: no cover
             try:
                 chat_id = user["chatId"]
                 exclude = self.dao.get_user_blocked_words(chat_id)
-                word_data = self.word_bank.get_random(exclude=exclude)
+                levels = self.dao.get_user_levels(chat_id)
+                word_data = self.word_bank.get_random(exclude=exclude, levels=levels)
+                msg: str = ''
 
-                msg: str = utils.build_word_msg(word_data)
+                if not word_data:
+                    msg = 'Du hast alles gelernt! - ¡Te lo has aprendido todo!'
+                else:
+                    msg = utils.build_word_msg(word_data)
```

Comment:
`msg =  utils.build_word_msg(word_data) if word_date else 'Du hast alles gelernt! - ¡Te lo has aprendido todo!'`

is a more pythonic way ;)